src/main/java/org/comic_conmuseum/fan_forge/backend/Application.java

```diff
@@ -101,11 +101,18 @@ private void addTestData() throws SQLException {
         for (int eIdx = 0; eIdx < exhibitTitles.size(); ++eIdx) {
             String title = exhibitTitles.get(eIdx);
             Instant exhibitMade = Instant.now().minus(eIdx + 5, ChronoUnit.DAYS);
+            // tags are based on even/odd indexes
             long exhibitId = exhibits.create(new Exhibit(
                     0, title, "Description for " + title, original.getId(),
                     exhibitMade, new String[] { "post", "exhibit", eIdx % 2 == 0 ? "even" : "odd", "index:" + eIdx },
-                    null
+                    null, false
             ), original);
+
+            // exhibit 1 is featured
+            if (exhibitId == 1) {
```

Comment:
I would prefer to have more than just one exhibit be featured (maybe `exhibitId % 10 == 1`, so we get two?) but that's just personal preference, and it's really not necessary.

Reply:
Why not?

Reply:
Exhibits 1 and 11 are now featured